extensions/wikia/ArticleVideo/scripts/article-featured-video.js

```diff
@@ -135,6 +135,15 @@ require(['wikia.window', 'wikia.onScroll', 'wikia.tracker', 'ooyala-player', 'wi
 			}
 		}
 
+		function initAttributionTracking() {
+			$('.attribution-container a').click(function () {
```

Comment:
$('.attribution-container').find('a') - do we care about optimization like this?